### PR TITLE
DWPF-1062,1595,1596: Extra information sent to GTM 

### PR DESCRIPTION
**Pull request by marcelkornblum**
- [x] JIRA ticket referenced in title
- [x] Title is clear and concise
- [x] Description gives any relevant detail
- [x] Tests are up to date
- [x] Documentation is up to date

Much more individual user data, page information and A/B test feature flag information added tot he GTM dataLayer on each page in the head, via the base template.

### CODE REVIEW

File: src/core/templates/includes/gtm_datalayer_info.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        "type": "{{ self.content_type.name|default_if_none:"NA" }}",
+        "tags": {
+            "topics": [{% for topic in self.topics.all %}"{{ topic.topic.title }}"{% if not forloop.last %},{% endif %}{% endfor %}],
+        }
+    },
+    "user_info": {
+        "profile_slug": "{{ request.user.profile.slug }}",
+        "roles": [{% for role in request.user.profile.roles.all %}
+            {
+                "team_slug: "{{ role.team.slug }}",
```
Comment (SamDudley) [inline src/core/templates/includes/gtm_datalayer_info.html:15]:
```suggestion
                "team_slug": "{{ role.team.slug }}",
```

**Review by SamDudley — APPROVED** [review]

**Review by SamDudley — APPROVED** [review]

**Review by CamLamb — APPROVED** [review]
LGTM! some clean Django'ness here!

**Comment by marcelkornblum** [comment]
Just waiting on a confirmation from Analytics that they can ingest this OK before merging

**Comment by marcelkornblum** [comment]
this will need updating now to access the feature flags via the dict, not directly

**Pull request merged**